src/plugins/saved_objects_management/public/management_section/object_view/components/header.tsx

```diff
@@ -71,8 +52,7 @@ export const Header = ({
           >
             <FormattedMessage
               id="savedObjectsManagement.view.deleteItemButtonLabel"
-              defaultMessage="Delete {title}"
-              values={{ title: type }}
+              defaultMessage="Delete"
```

Comment:
NIT: now that we're providing the `title` to the component, we would be able to put it back in this message. Not sure this is really better though, a plain `Delete` wording may be better.